src/nuxeo-cloud/what-is-nuxeo-cloud.md

```diff
@@ -0,0 +1,208 @@
+---
+title: What's Nuxeo Cloud
+review:
+    comment: ''
+    date: '19-03-2020'
+    status: ok
+labels:
+    - nuxeo-cloud
+tree_item_index: 100
+toc: true
+---
+
+Nuxeo Cloud is a fully hosted and managed offering providing our customers with setup, support and maintenance for a complete Nuxeo implementation.
+
+This includes support and management of the infrastructure, third-party services, and application, including all deployments and/or changes to production and pre-production environments.
+
+![]({{file name='cloud-architecture.png'}})
+
+## Nuxeo Cloud Environments
+
+The Nuxeo Cloud Operations (NCO) team provides different environments as part of a standard configuration with the latest LTS and Hotfixes applied.
+
+### Production
+
+Primary environment used by customers:
```

Comment:
change this to 

Production environments are the primary environment used by all Nuxeo Cloud customers. Production environments include: 

- Support and management of all changes and deployments by the Nuxeo Cloud team
- Complete monitoring and 24x7 Support by the Nuxeo Cloud team
- PCI/DSS and SOC2 Certified (as of 2019)